circlecheck.lic

```diff
@@ -500,12 +498,24 @@ $GuildReqs = {
 }
 
 def main
-  level_target = [variable.find { |x| x =~ /\d+/ }.to_i, DRStats.circle + 1].max
+  arg_definitions = [
+    [
+      { name: 'debug', regex: /debug/i, optional: true },
+      { name: 'brief', options: %w(brief short next), optional: true, description: 'Only display requirements you don\'t meet' },
```

Comment:
Since 'brief' is one of the options, I think it should not also be its name. Maybe 'mode' or 'output'?